Fix:
error: implicit declaration of function 'yylex' [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'yyerror' [-Werror,-Wimplicit-function-declaration]
--- src/aclyacc.y.orig	2010-08-26 07:33:35.000000000 -0500
+++ src/aclyacc.y	2022-01-17 02:38:19.000000000 -0600
@@ -3,6 +3,9 @@
 #include "acl2.h"
 #include <stdio.h> /* XXX REMOVE */
 
+int yylex(void);
+void yyerror(const char *);
+
 /* XXX remove */
 unsigned char fmt_buf[32];
 unsigned char fmt_buf2[32];
